pages/linux/gs.md

```diff
@@ -0,0 +1,57 @@
+# gs
+
+> GhostScript is a PDF and PostScript interpreter.
+
+- To view a file:
+
+`gs -dQUIET -dBATCH {{file.pdf}}`
+
+- Reduce PDF file size to 150 dpi images for reading on a ebook device:
+
+`gs -dNOPAUSE -dQUIET -dBATCH -sDEVICE=pdfwrite -dPDFSETTINGS=/ebook -sOutputFile={{output.pdf}} {{input.pdf}}`
+
+- Convert PDF file to an image:
+
+`gs -dQUIET -dBATCH -dNOPAUSE -sDEVICE=jpeg -r150 -dTextAlphaBits=4 -dGraphicsAlphaBits=4 -dFirstPage=1 -dLastPage=3 -sOutputFile={{output_%d.jpg}} {{input.pdf}}`
+
+- Extract pages from a PDF file:
+
+`gs -dQUIET -dBATCH -dNOPAUSE -sDEVICE=pdfwrite -dFirstPage=1 -dLastPage=3 -sOutputFile={{output.pdf}} {{input.pdf}}`
+
+- Merge PDF files:
+
+`gs -dQUIET -dBATCH -dNOPAUSE -sDEVICE=pdfwrite -sOutputFile={{output.pdf}} {{input1.pdf}} {{input2.pdf}}`
+
+- Convert from PostScript file to PDF file:
+
+`gs -dQUIET -dBATCH -dNOPAUSE -sDEVICE=pdfwrite -sOutputFile={{output.pdf}} {{input.ps}}`
+
+<!---
```

Comment:
Could you add this to the PR description?

Reply:
by this you mean the comment?

Reply:
Yes. It's better to have this as part of the OR description, that way we can read it separately and it won't cause the parser to fail (comments aren't supported). :+1:

I'd say the examples you've provided seem fine. We can then review the syntaxing of the page, and provide any feedback. :+1:

Reply:
done